pages/vi/vi-planet-installation-and-configuration.md

```diff
@@ -125,6 +125,9 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
   - `docker tag treehouses/planet:latest treehouses/planet:local`
   - `docker tag treehouses/planet:db-init treehouses/planet:db-init-local` 
 
+6. To exit the logs press `CTRL+C` when you see the following code: 
+![db-init-img](https://user-images.githubusercontent.com/41874655/44394481-03d79d00-a557-11e8-91ed-378eccc95011.jpg)
```

Comment:
Hi, @janamr looks great! However, probably better to store image inside 'images' folder of the project and have a link like this: `![db-init-img](images/vi-db-init.png)`.
![a5f466497ac4e67aca5dfd130b697c937db5b06c1a957a9bc4 pimgpsh_fullsize_distr](https://user-images.githubusercontent.com/35533801/46684790-da83d680-cbc1-11e8-8ad1-ab2acf4f3cb5.png)

Also, you could create an empty line after '....the following code:' before image, in that way number 6 in the list will not appear dislocated.
![1](https://user-images.githubusercontent.com/35533801/46684824-ed96a680-cbc1-11e8-99ce-e0a9902f50dc.jpg)

Reply:
Hey thanks @svlesiv I was wondering about that. I'll fix it now.

Reply:
@svlesiv I agree with both points - thanks